samples/notebooks/python/09-streaming-completions.ipynb

```diff
@@ -0,0 +1,249 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "68e1c158",
+   "metadata": {},
+   "source": [
+    "# Streaming Results"
```

Comment:
Add at the top of the notebook something like:

"In this notebook, we show how you can stream the output of Completion models with the Semantic Kernel. Previously, both text and chat completion models would return the entire response as a giant block of text. With this, you'll be able to stream the output of the response as the response is completing which can help provide a more natural chatting experience."